homeassistant/components/hyundai_kia_connect/translations/en.json

```diff
@@ -0,0 +1,26 @@
+{
+    "title": "Hyundai / Kia Connect",
```

Comment:
This file doesn't match `strings.json`? Please update the translations using the translation tools.